dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task220825MakeSomeSystemFieldsRemovableTest.java

```diff
@@ -0,0 +1,37 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotcms.util.IntegrationTestInitService;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Verifies that the {@link Task220825MakeSomeSystemFieldsRemovable} UT runs as expected.
+ *
+ * @author Jose Castro
+ * @since Aug 30th, 2022
+ */
+public class Task220825MakeSomeSystemFieldsRemovableTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
+        // Setting up the web app environment
+        IntegrationTestInitService.getInstance().init();
+    }
+
+    @Test
+    public void testExecuteUpgrade() {
+        // This UT updates system Content Types, so no test data is required
+        final Task220825MakeSomeSystemFieldsRemovable upgradeTask = new Task220825MakeSomeSystemFieldsRemovable();
+        assertTrue("One or more of the Content Types whose fields need to be updated don't match the expected ID",
+                upgradeTask.forceRun());
+        try {
+            upgradeTask.executeUpgrade();
```

Comment:
Shouldn't we check here that after the task runs the base types have the expected values for the columns?

Reply:
Done!